opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -239,7 +239,7 @@ namespace Opm {
                     }
 
                     // add trivial equation for 2p cases (Only support water + oil)
```

Comment:
Just curious: I assume there is some kind of checking that we do not try to run this with (oil + gas) or (water + gas)?